cmd/neofs-cli/modules/container/create.go

```diff
@@ -150,10 +150,9 @@ func initContainerCreateCmd() {
 	flags.DurationP(commonflags.Timeout, commonflags.TimeoutShorthand, commonflags.TimeoutDefault, commonflags.TimeoutUsage)
 	flags.StringP(commonflags.WalletPath, commonflags.WalletPathShorthand, commonflags.WalletPathDefault, commonflags.WalletPathUsage)
 	flags.StringP(commonflags.Account, commonflags.AccountShorthand, commonflags.AccountDefault, commonflags.AccountUsage)
-
-	flags.StringVar(&containerACL, "basic-acl", acl.NamePrivate, fmt.Sprintf("HEX encoded basic ACL value or keywords like '%s', '%s', '%s'",
-		acl.NamePublicRW, acl.NamePrivate, acl.NamePublicROExtended,
-	))
+	flags.StringVar(&containerACL, "basic-acl", acl.NamePrivate, fmt.Sprintf("HEX encoded basic ACL value or one of the keywords ['%s', '%s', '%s',\n '%s', '%s', '%s', '%s', '%s]'\n"+
```

Comment:
Why `\n` in the string? I don't think we should try formatting it here.

Also, "HEX-encoded".

Reply:
You don't know the width of the target terminal, so these newlines just can't be placed correctly for everyone. So we either use some shorter strings with references (see smth), or don't care about their length.